inlong-manager/manager-workflow/src/main/java/org/apache/inlong/manager/workflow/event/process/ProcessEventListener.java

```diff
@@ -32,4 +38,16 @@ public interface ProcessEventListener extends EventListener<ProcessEvent> {
      */
     List<ProcessEventListener> EMPTY_LIST = Lists.newArrayList();
 
+    /**
+     * Async process common thread pool
+     */
+    ExecutorService executorService = new ThreadPoolExecutor(
```

Comment:
The variables in the interface are `public static final`, please change the name to the big camel case, thanks.